### PR TITLE
Feature/render country info

### PR DESCRIPTION
**Pull request by huasmc**
EVERYTHING

### CODE REVIEW

File: client/src/views/country_info.js [hunk 1]
```diff
@@ -3,9 +3,13 @@ const CountryInfo = function () {
 }
 
 CountryInfo.prototype.render = function (country) {
-  // TODO: Render country info
```
Comment (huasmc) [inline client/src/views/country_info.js:6]: REMOVE IT

File: .gitignore [hunk 2]
```diff
@@ -1 +1,3 @@
 /node_modules
+bundle.js
```
Comment (alinemokfa) [inline .gitignore:2]: Oops. I didn't mean to type this. Don't approve it.

File: client/src/views/country_info.js (hunk 1 above)
Comment (alinemokfa) [reply client/src/views/country_info.js:6]: Be nicer! ;)

Comment (alinemokfa) [reply client/src/views/country_info.js:6]: Haha

**Pull request merged**